### ISSUE TITLE
Creating and updating a theme has many steps 

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/accessible-pygments#34]
Working on #33, this is the full set of steps I had to do in order to feel like I fully updated the repo when I updated a theme:

1. Update the colors in the `style.py` file
2. `hatch run dev:create_css`
3. `hatch run dev:render_html`
4. Pick and choose which updated files to `git add`, then `git restore` the irrelevant ones
5. Update the theme's README file, which requires manually calculating new contrast values for the table in the README
6. Open one of the example files under `test/results/<theme-name>/` in my browser, take a screenshot, save it under `a11y_pygments/<theme-name>/images/<theme-name>.png`, and then update the link in the theme's README.

Seems to me that more of this could be automated, or at least better documented. 

I also get the feeling that some of the file changes could be simplified. It looks like duplicate generated files are getting checked into the repo, for example:

- [a11y_pygments/a11y_high_contrast_light/style.css](https://github.com/Quansight-Labs/accessible-pygments/pull/33/files#diff-7f0b69d5dd0bce20169594c4b7f93f590ef3deebe609fc63b8d2038cdd1ea011)
- [docs/css/a11y_high_contrast_light-style.css](https://github.com/Quansight-Labs/accessible-pygments/pull/33/files#diff-d2bf6a45fdb78c6c46efae2d2480a420da774d06b186ff6fcffd89f761dad0f1)

**Comment by trallard** [comment]
yes! I have been wanting to automate a lot of these steps because there is indeed a looooot to do.
